.travis/linux..install.sh

```diff
@@ -1,25 +1,25 @@
 #!/usr/bin/env bash
 
-PACKAGES="cmake libsndfile-dev fftw3-dev libvorbis-dev libogg-dev libmp3lame-dev
-	libasound2-dev libjack-jackd2-dev libsdl-dev libsamplerate0-dev libstk0-dev stk
-	libfluidsynth-dev portaudio19-dev g++-multilib libfltk1.3-dev
-	libgig-dev libsoundio-dev"
```

Comment:
Please don't rearrange these.  The less lines we change, the easier this is to merge with other PRs, etc.

Reply:
What other PRs? Sorting this list is an improvement. One package per line would be even better.

Reply:
>Perhaps but it is out of scope.

I propose it now. Do you agree?

Reply:
I'd say let's do that on `master` once we merge `stable-1.2` into it. It will reduce the merge conflicts.